docs/5-development/09-internationalization.md

```diff
@@ -1 +1,66 @@
-# Internationalization
\ No newline at end of file
+# Internationalization (i18n)
+
+In this article, we will discuss internationalization (i18n) in the context of UI5 Web Components.
+
+Internationalization is crucial for creating applications that support multiple languages and regional settings. The UI5 Web Components library provides an easy way to internationalize your components using i18n files.
+
+## i18n Files
+
+i18n files define translatable texts as key-value pairs, separated by `=` in the `messagebundle.properties` file. Translations for multiple languages can be provided by creating additional properties files.
+
+To define translatable texts, create a `src/i18n/messagebundle.properties` file with key-value pairs. For example:
+
+```
+# please wait text for the sample component
+PLEASE_WAIT=wait
+```
+
+**Note:** Comments for translatable text can be added using the `#` symbol.
+
+To create key-value pairs in a specific language different from the default one, create files following this pattern: `messagebundle_{locale}.properties`. For example:
+- **Translations in German:** `src/i18n/messagebundle_de.properties`
+- **Translations in English:** `src/i18n/messagebundle_en.properties`
+
+Here is an example of providing translations:
+
+```properties
+# Spanish translation
+PLEASE_WAIT=Espere
+```
+
+## Integrating i18n into Components
+
+To integrate internationalization (i18n) into UI5 web components, you need to load the i18n bundle that contains all translatable texts for the current package.
```

Comment:
either `web components` in general or `UI5 Web Components`, `UI5 web components` seems wrong